tools/nni_cmd/nnictl_utils.py

```diff
@@ -482,25 +461,25 @@ def export_trials_data(args):
         return
     running, response = check_rest_server_quick(rest_port)
     if running:
-        response = rest_get(trial_jobs_url(rest_port), 20)
+        response = rest_get(export_data_url(rest_port), 20)
         if response is not None and check_response(response):
-            content = json.loads(response.text)
-            # dframe = pd.DataFrame.from_records([parse_trial_data(t_data) for t_data in content])
-            # dframe.to_csv(args.csv_path, sep='\t')
-            records = parse_trial_data(content)
             if args.type == 'json':
-                json_records = []
-                for trial in records:
-                    value = trial.pop('reward', None)
-                    trial_id =  trial.pop('id', None)
-                    json_records.append({'parameter': trial, 'value': value, 'id': trial_id})
-            with open(args.path, 'w') as file:
-                if args.type == 'csv':
-                    writer = csv.DictWriter(file, set.union(*[set(r.keys()) for r in records]))
+                f_obj = open(args.path, 'w')
+                f_obj.write(response.text)
+                f_obj.close()
+            else:
```

Comment:
suggest using
```python
elif args.type=='csv':
...
else:
    raise ValueError("export file type not supported, please use json or csv instead")
```
 here

Reply:
good

Reply:
Do not suggest to use raise ValueError(''), it is not friendly for users in command line terminal. Use print_error('') instead.

Reply:
@SparkSnail thanks!

Reply:
> suggest using
> 
> ```python
> elif args.type=='csv':
> ...
> else:
>     raise ValueError("export file type not supported, please use json or csv instead")
> ```
> 
> here

Do not suggest to use `raise ValueError('')`, it is not friendly for users in command line terminal. Use `print_error('')` and `exit(1)` instead.